src/server/database/migrations/20180226074116-meetingSettings.js

```diff
@@ -39,7 +38,7 @@ exports.up = async (r) => {
           id: shortid.generate(),
           meetingType: RETROSPECTIVE,
           teamId,
-          phases: [LOBBY, CHECKIN, THINK, GROUP, VOTE, DISCUSS, SUMMARY]
+          phases: [LOBBY, CHECKIN, 'think', GROUP, VOTE, DISCUSS, SUMMARY]
```

Comment:
Should this be a constant called `REFLECT` instead?

Reply:
nah, can't go changing history! i wanted to keep the previous migration immutable, but also get rid of the THINK constant.
the latest migration does the thing you're after: https://github.com/ParabolInc/action/blob/6af1d5aa3b19845da0e7d79a840c3720b329d132/src/server/database/migrations/20180319122615-retroReflection.js#L27